include/itkSpectra1DImageFilter.hxx

```diff
@@ -144,10 +149,10 @@ Spectra1DImageFilter<TInputImage, TSupportWindowImage, TOutputImage>::ComputeSpe
   const double overlap = 0.5;
   IndexType    segmentIndex(lineIndex);
   const double spectralScale = 1.0 / (fftSize * fftSize);
-  for (unsigned int segment = 0; segment < 3; ++segment)
+  for (unsigned int segment = 0; segment < 3; ++segment) // Why 3?
```

Comment:
@thewtex @dzenanz Could you please clarify how segments are being used here? Why are we using 3?